wire/modules/Fieldtype/FieldtypeOptions/FieldtypeOptions.module

```diff
@@ -171,8 +171,9 @@ class FieldtypeOptions extends FieldtypeMulti implements Module {
 			
 		} else if(is_string($value)) {
 			// may be option 'title' (first) or option 'value' (second)
+			$_value = $value;
 			$value = $this->manager->getOptions($field, array('title' => $value)); 
```

Comment:
This line returned a `SelectableOptionArray`, which was used by this line ([SelectableOptionManager.php#L132](https://github.com/ryancramerdesign/ProcessWire/blob/master/wire/modules/Fieldtype/FieldtypeOptions/SelectableOptionManager.php#L132)) as array key, as soon as the next line called `getOptions` again. Using objects as array keys doesn't work, so it failed.